docs/user/boot_options/index.md

```diff
@@ -32,6 +32,22 @@ to extend.
   agama.config_url=usb:///agama.yaml
   ```
 
+- `agama.info`
+  Points agama to info file that contains additional agama settings. It is useful when more options
+  are provided to agama and typing during boot is unreliable.
+
+  ```text
+  agama.info=ftp://myserver/devel.info
+  ```
+
+  :::warning
+  Available from Agama12.
+  :::
+
+  :::warning
+  Supports only parameters specified on this section. Kernel parameters are ignored.
```

Comment:
I do not think it is a warning. You can make it part of the description of the options or, if you want to emphasize this, let's use a simple `note`.